src/lib/exec-fetcher.test.js

```diff
@@ -0,0 +1,70 @@
+const test = require('ava')
+const path = require('path')
+const fs = require('fs-extra')
+const tempDirectory = require('temp-dir')
+const { v4: uuid } = require('uuid')
+const { getExecName, shouldFetchLatestVersion, fetchLatestVersion } = require('./exec-fetcher')
+
+test.beforeEach(t => {
+  const directory = path.join(tempDirectory, `netlify-cli-exec-fetcher`, uuid())
+  t.context.binPath = directory
+})
+
+test(`should postix exec with .exe on windows`, t => {
+  const execName = 'some-binary-file'
+  if (process.platform === 'win32') {
+    t.is(getExecName({ execName }), `${execName}.exe`)
+  } else {
+    t.is(getExecName({ execName }), execName)
+  }
+})
+
+const packages = [
```

Comment:
Feels a bit strange to have this here and not in a dedicated test file (one for the tunnel client and one for the traffic mesh).
However I prefer it this way since we're actually testing the `exec-fetcher`